mne/io/curry/curry.py

```diff
@@ -294,12 +291,17 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
         """Read a chunk of raw data."""
         if self._is_ascii:
             ch_idx = range(0, len(self.ch_names))
-            if version.parse(np.version.version) >= version.parse("1.16.0"):
+            # mne doesn't have packaging lib, so we check np.version like this
+            np_vers = np.array(np.version.version.split("."), dtype=int)
```

Comment:
I would have used https://github.com/mne-tools/mne-python/blob/master/mne/utils/_testing.py#L188

Reply:
Good to know, Thanks!

I'll put that in